src/doc/rustc/src/platform-support/riscv64a23-unknown-linux-gnu.md

```diff
@@ -0,0 +1,66 @@
+# `riscv64a23-unknown-linux-gnu`
+
+**Tier: 3**
+
+RISC-V targets using the ratified [RVA23 Profile](https://github.com/riscv/riscv-profiles/blob/main/rva23-profile.adoc).
+This target will enable all mandary features of rva23u64 and rva23s64 by default.
+
+
+## Target maintainers
+
+[@ZhongyaoChen](https://github.com/ZhongyaoChen)
+[@CaiWeiran](https://github.com/CaiWeiran)
+
+## Requirements
+
+This target requires:
+
+* Linux Kernel version 4.20 or later
+* glibc 2.17 or later
+
+
+## Building the target
+
+The target is distributed through `rustup`, and otherwise require no
+special configuration.
+
+If you need to build your own Rust for some reason though, the target can be build with:
+
+```bash
+./x build --target riscv64a23-unknown-linux-gnu
+```
+
+## Building Rust programs
+
+Add the target:
+
+```bash
+rustup target add riscv64a23-unknown-linux-gnu
+```
```

Comment:
Tier 3 targets are usually not distributed via rustup, right?

Reply:
Correct, this instruction would not work.

Reply:
No, the target docs should accurately reflect the current reality, not the future intent. Remove it for now, you can add it when it's tier 2.